NitroxClient/GameLogic/InitialSync/PostInitialSyncProcessor.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.InitialSync.Base;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.InitialSync;
+
+/// <summary>
+/// Happens after each initial sync processor finished loading
+/// </summary>
+public class PostInitialSyncProcessor : InitialSyncProcessor
```

Comment:
Imo there should not be a general `PostInitialSyncProcessor` for a concrete case. If the ping logic can't fit in existing `InitialSyncProcessor` then create a new one with concrete `DependentProcessors`.